modules/emulator/R/calcSpatialCov.R

```diff
@@ -2,5 +2,8 @@
 ##' @title calcSpatialCov 
 ##' @export
 ##'
+##' @param x Name of variable to plot on X axis
```

Comment:
Likewise, X here isn't a plot. If you look at the two options that get called (calcSpatialCov.list and calcSpatialCov.matrix) the first argument is either a spatial distance matrix or a list of component spatial distance matrices